src/model/kaniRunner.ts

```diff
@@ -136,6 +150,17 @@ function executeKaniProcess(
 ): Promise<any> {
 	return new Promise((resolve, reject) => {
 		execFile(kaniBinaryPath, args, options, (error, stdout, stderr) => {
+			// Store the output of the process into an object
+			const output: CommandOutput = {
+				stdout: stdout.toString().trim(),
+				stderr: stderr.toString().trim(),
```

Comment:
Do we need `.trim()` here? I'd refrain from using it and just emit the output "as is" to avoid non-matching outputs if we need to debug the output at some point.